app/locale/uk.coffee

```diff
@@ -1182,7 +1182,7 @@ module.exports = nativeDescription: "Українська", englishDescription:
     join_class: "Приєднайтесь до класу"
     join_class_2: "Приєднайтесь до класу"
     ask_teacher_for_code: "Запитайте у вашого вчителя, чи є у вас код класу CodeCombat! Якщо так, введіть його нижче:"
-    enter_c_code: "<Введіть код класу>"
+    enter_c_code: "<Уведіть код класу>"
```

Comment:
It should be `уведіть` because it's easer to read. It's hard to pronounce this **вв**. Don't you think so? It's commonly used because it's calque from russian.
But if you think `введіть` is better I can roll it back.